src/core.js

```diff
@@ -1140,7 +1140,11 @@ const _makeWindow = (options = {}, parent = null, top = null) => {
       super(parts && parts.map(part => utils._normalizePrototype(part, global)), opts);
     }
   })(Blob);
-  window.FormData = FormData;
+  window.FormData = (Old => class FormData extends Old {
+    append(field, value, options) {
+      super.append(field, utils._normalizePrototype(value, global), options);
+    }
+  })(window.FormData);
```

Comment:
Should this be `FormData` and not `window.FormData`

Reply:
https://github.com/webmixedreality/exokit/pull/628/commits/faf22e416cf6edc8f34fd94b10da659e85c49b71